docs/detections/detection-engine-intro.asciidoc

```diff
@@ -5,6 +5,7 @@
 
 beta[]
```

Comment:
Hey guys @nastasha-solomon and @DonNateR the other merged PR #519 had the `beta[]` tag removed. Please make sure we do that here. #519 was not yet backported -- don't know if that was intentional or not, but when this is merged please backport to 7.12, 7.x, 7.11. Will be good practice for Nastasha. Thanks!

Reply:
@jmikell821: That PR was actually closed, not merged. We'll likely merge this one instead.